calculator-service/src/main/java/de/viadee/camunda/connected/calculator/externaltaskworker/ExternalTaskCalculatorHandler.java

```diff
@@ -0,0 +1,65 @@
+package de.viadee.camunda.connected.calculator.externaltaskworker;
+
+import java.util.Collections;
+import java.util.logging.Logger;
+
+import javax.annotation.PostConstruct;
+
+import org.camunda.bpm.client.ExternalTaskClient;
+import org.camunda.bpm.client.task.ExternalTask;
+import org.camunda.bpm.client.task.ExternalTaskHandler;
+import org.camunda.bpm.client.task.ExternalTaskService;
+import org.camunda.bpm.engine.variable.Variables;
+import org.camunda.bpm.engine.variable.value.ObjectValue;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import de.viadee.camunda.connected.calculator.service.CalculatorService;
+
+@Component
+public class ExternalTaskCalculatorHandler implements ExternalTaskHandler {
+	
+	private final static Logger LOGGER = Logger.getLogger(ExternalTaskCalculatorHandler.class.getName());
+	
+	@Autowired
+	private ExternalTaskClient externalTaskClient;
+	
+	@Autowired
+	private CalculatorService calculatorService;
+	
+	@Value("${lockDuration}")
+	private int lockduration;
+	
+	@Value("${topicSubscription}")
+	private String topicSubscription;
+
+	@Override
+	public void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
+
+		long num1x = (long) externalTask.getVariable("num1");
```

Comment:
better we use getVariableTyped from the camunda-API?

Reply:
why not simply use `long num1x = externalTask.getVariable("num1");` without casting, as the method is already generic?